apertium_apy/utils/translation.py

```diff
@@ -318,8 +321,9 @@ def translate_pipeline(to_translate, commands):
         all_cmds.append(cmd)
 
     proc_reformat = Popen('apertium-rehtml-noent', stdin=PIPE, stdout=PIPE)
+    assert proc_reformat.stdin is not None  # stupid mypy
     proc_reformat.stdin.write(towrite)
-    towrite = proc_reformat.communicate()[0].decode('utf-8')
+    towrite = proc_reformat.communicate()[0].decode('utf-8').encode('utf-8')
```

Comment:
Can we just... not decode in the first place?

Reply:
doh! fixed in c2fa306857c929c1bdba33580672e32270e4350c